### ISSUE TITLE
SplitTheBill

### ISSUE THREAD

**Issue by imtiaz-ak** [DevArifHossain/saas-idea-validator#9]
**Problem**
When teenagers/students with not a lot of money hangout at restaurants and cafes, they often split the bill according to what they ordered. When paying, someone has to bring out a calculator and do the accounting. This gets very messy when the group is large enough. A lot of the times there's an error in the accounting and as a result there's a shortage of money despite everyone paying. Someone has to pay extra to pay the bills. The fact that restaurants put the pre-VAT+SD price on the menu and then adds VAT and SD on the receipt doesn't help either.

**Solution**
SplitTheBill could be an app where you enter the items and their price and then choose who ate what. The app does all the calculation and gives you a receipt of who needs to pay how much. No one wants to do accounting when they're hanging out. So, you delegate that job to your phone, who doesn't make mistakes either. The process could also be made simpler by adding popular restaurants and their menus on the app so the users have to only choose who ate what.

**Monetization**
I would have data on the eating habits of users. This would allow me to run very targeted ads. These ad spots could be sold to restaurants for $$

**Comment by DevArifHossain** [comment]
There's many existing bill splitting apps but your monetizetion system seems to be different. 
I wander how will you know about user's eating habits? Like they will enter the dish name?

**Comment by imtiaz-ak** [comment]
Yes. If the restaurant's menu is already in the app, users choose who ate what, letting me know the type of food this user and their friends eat. If the restaurant's menu is not on the app, they will have to type the name and price of the items, which will also give me an idea of the type of food the user is into.

If anyone frequently has burgers from madchef, chillox etc. I know that user and their friend circle likes burgers and similar fast foods. A newer burger place could get very high roi by showing ads specifically to these burger-lovers.

**Comment by DevArifHossain** [comment]
Yeap. Makes sense 🚀

**Comment by omranjamal** [comment]
Entering this info will be a PITA though. Are you considering a bit of Machine Vision into the mix?

**Comment by imtiaz-ak** [comment]
I agree. But doing the accounting is an even bigger PITA. Making it a lesser pain will be the biggest challenge for this app imo. Using data and machine learning or even visual AI are solutions that I would have to try through iterations.

**Comment by NafishaNawrin** [comment]
There can also be something similar like split the rent. A lot of bachelors have similar problems like owing someone money or vice versa for the rents, utility fees, electricity gas bills etc. They will be able to know who needs to pay how much to whom here as well.
